locales/en-GB/messages.properties

```diff
@@ -33,6 +33,7 @@ errorLoadingProjectSuggestion=NOTE: if you are using Private Browsing mode, plea
 errorUnsupportedBrowser=Sorry, Thimble won't work in your Browser
 errorUnsupportedBrowserSuggestion=Thimble works in all of the following browsers: <a title="Download Mozilla Firefox" href="https://www.mozilla.org/{{ locale }}/firefox/new/">Firefox</a>, <a title="Download Google Chrome" href="https://www.google.com/chrome/browser/desktop/index.html">Chrome</a>, Internet Explorer 11, Microsoft Edge, Safari (8+), and <a title="Download Opera" href="http://www.opera.com/download">Opera</a>.
 errorUnsupportedBrowserIgnore=Let me try anyway!
+errorLoadingProjectBramble=For error details open your browser's Developer Tools console.
```

Comment:
locale strings should only be changed in `en-US`, every other locale gets overwritten based on that so your changes here will be lost when you sync locales.